facts/4.11/centos-10-x86_64.facts

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
did you regenerate this factset intentionally? We usually try to avoid regenerating factsets unless they have some bugs. It just creates a lot of noise in the git history.

Reply:
nope, this cam automatically, i din't enforce this.... also wondering why i got this change

Reply:
ah only for the gem based factsets we've a check to not regenerate them: https://github.com/voxpupuli/facterdb/blob/master/facts/get_facts.sh#L232

Can you please undo the changes in 4.11/centos-10-x86_64.facts for now?

Reply:
okay, will do